lib/reporter/global-reporter.js

```diff
@@ -106,28 +106,34 @@ module.exports = class GlobalReporter {
     return this;
   }
 
-  loadFile() {
-    const builtInReporterFileName = path.join(__dirname, 'reporters', this.reporterFile + '.js');
-
-    return Utils
-      .fileExists(builtInReporterFileName)
-      .then(fileExists => {
-        if (!fileExists) {
-          return this.loadCustomReporter();
-        }
+  loadReporter() {
+    if (Array.isArray(this.reporterFile)) {
+      return Promise.all(this.reporterFile.map(file => this.loadFile(file)));
+    } else {
+      return this.loadFile(this.reporterFile);
+    }
+  }
 
-        return require(builtInReporterFileName);
-      });
+  loadFile(reporterFile) {
+    const builtInReporterFileName = path.join(__dirname, 'reporters', reporterFile + '.js');
+
+    return Utils.fileExists(builtInReporterFileName).then((fileExists) => {
+      if (!fileExists) {
+        return this.loadCustomReporter(reporterFile);
+      }
+
+      return require(builtInReporterFileName);
+    });
```

Comment:
@gravityvi  This takes care of both built-in reporter and custom reporter

Reply:
Yes, but it generates a single report with one `--reporter` CLI value. Could we update it to generate the default `builtInReporter` every time along with the reporter mentioned via CLI

Reply:
yup, we can do that